apps/api/v2/src/modules/auth/strategies/access-token/access-token.strategy.ts

```diff
@@ -25,19 +25,15 @@ export class AccessTokenStrategy extends PassportStrategy(BaseStrategy, "access-
       const accessToken = request.get("Authorization")?.replace("Bearer ", "");
 
       if (!accessToken) {
-        throw new UnauthorizedException("Access token is missing or invalid.");
+        throw new UnauthorizedException("Access token is missing");
       }
 
-      const valid = this.oauthFlowService.validateAccessToken(accessToken);
-
-      if (!valid) {
-        throw new UnauthorizedException("Access token is missing or invalid.");
-      }
+      this.oauthFlowService.validateAccessToken(accessToken);
 
       const ownerId = await this.tokensRepository.getAccessTokenOwnerId(accessToken);
 
       if (!ownerId) {
-        throw new UnauthorizedException("Invalid access token");
+        throw new InternalServerErrorException("Access token not found in the database");
```

Comment:
NIT: what if we modify the error message as `Access token not found` instead of `Access token not found in the database`? But I don't understand why you can't use previous message `Invalid access token` because this sounds pretty clear and concise too so why change it?
Cc @ThyMinimalDev

Reply:
Now that the access token expiry has a different message `ACCESS_TOKEN_IS_EXPIRED` I think just returning `Invalid access token` should be good!